src/app/loadout/mod-utils.ts

```diff
@@ -12,12 +11,35 @@ import mutuallyExclusiveMods from 'data/d2/mutually-exclusive-mods.json';
 import { normalToReducedMod, reducedToNormalMod } from 'data/d2/reduced-cost-mod-mappings';
 import { knownModPlugCategoryHashes } from './known-values';
 
-export const plugCategoryHashToBucketHash: LookupTable<PlugCategoryHashes, BucketHashes> = {
+const armorPlugCategoryHashToBucketHash = {
   [armor2PlugCategoryHashesByName.helmet]: BucketHashes.Helmet,
   [armor2PlugCategoryHashesByName.gauntlets]: BucketHashes.Gauntlets,
   [armor2PlugCategoryHashesByName.chest]: BucketHashes.ChestArmor,
   [armor2PlugCategoryHashesByName.leg]: BucketHashes.LegArmor,
   [armor2PlugCategoryHashesByName.classitem]: BucketHashes.ClassArmor,
+} as const;
+
+export type ArmorBucketHashes =
+  (typeof armorPlugCategoryHashToBucketHash)[keyof typeof armorPlugCategoryHashToBucketHash];
+
+export const OrderedArmorBucketHashes: ArmorBucketHashes[] = [
+  BucketHashes.Helmet,
+  BucketHashes.Gauntlets,
+  BucketHashes.ChestArmor,
+  BucketHashes.LegArmor,
+  BucketHashes.ClassArmor,
+];
```

Comment:
We don't use CamelCase for values.

We have some very similar types already in `loadout-builder` and crossing this type bridge is always a bit of a hassle - i'd prefer if we didn't over-type the armor PCHs/BucketHashes here unless we also did it in the rest of `loadout/mod-*` and used a common set of types, not duplicates.

Reply:
My initial thought was to use the [D2Categories](https://github.com/DestinyItemManager/DIM/blob/master/src/app/destiny2/d2-bucket-categories.ts#L14-L20) to get this list. I ran into some annoying type issues doing that. But I'll give it another shot.

Reply:
Ripped out both of these things.